MoltenVK/MoltenVK/GPUObjects/MVKImage.mm

```diff
@@ -533,7 +536,7 @@
 
 VkDeviceSize MVKImage::getBytesPerLayer(uint8_t planeIndex, uint32_t mipLevel) {
     VkExtent3D extent = getExtent3D(planeIndex, mipLevel);
-    size_t bytesPerRow = getPixelFormats()->getBytesPerRow(_vkFormat, extent.width);
+    size_t bytesPerRow = mvkAlignByteCount(getPixelFormats()->getBytesPerRow(_vkFormat, extent.width), _rowByteAlignment);
```

Comment:
Could this line be consolidated as:

    size_t bytesPerRow = getBytesPerRow(planeIndex, mipLevel);

which would consolidate format and alignment management into `getBytesPerRow()`. That might reduce fragility moving forward.

Reply:
I have updated the code as you requested.